src/sentry/static/sentry/app/components/globalModal.jsx

```diff
@@ -86,7 +87,7 @@ const GlobalModalContainer = createReactClass({
 
     return (
       <GlobalModal {...this.props} {...modalStore} visible={visible}>
-        {visible && modalStore.renderer}
+        {visible ? modalStore.renderer : null}
```

Comment:
I don't like that we have to switch these, ternaries are IMO harder to read and easier to mess up. have no solution though. 🤷‍♂️

Reply:
I don't think it's too bad, especially since ternaries are used pretty often already.